src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -50,6 +50,25 @@ export default class SimidMapCreative extends BaseSimidCreative {
    * @private 
   */
   grantLocationAccess_() {
-    //ToDo(kristenmason@): implement map
+    this.loadMap_();
+  }
+
+  /**
+   * Loads a map object that currently displays a hardcoded location.
+   * @param {Object} coordinates - the LatLng object of user's current location
```

Comment:
minor nit.
Remove the `-` and capitalize "The"

Reply:
done